src/ecrc-api/src/main/resources/application.properties

```diff
@@ -1,6 +1,18 @@
 server.servlet.context-path=/ecrc
 server.port=8082
 
+#Actuator Configuration
+management.endpoints.enabled-by-default=false
+management.endpoint.info.enabled=true
+management.endpoint.health.enabled=true
+management.endpoint.health.show-details=always
+management.endpoint.health.show-components=always
+
+#Actuator Info Endpoint Configuration
+info.app.name=eCRC
+info.app.description=Ministry of Attorney General - eCRC
+info.app.version=1.0.0
```

Comment:
Could you change this to: @project.version@ also the others should pull these values from the pom as well.